docs/adding-or-modifying-a-component.md

```diff
@@ -0,0 +1,43 @@
+# Add or modify components in SDK 7
+
+Adding a new component to the SDK is an important change, because it will need to be supported in the future. This also must contemplaye avoiding breaking existing components by mistake.
```

Comment:
```suggestion
Adding a new component to the SDK is an important change, because it will need to be supported in the future. This also must contemplate avoiding breaking existing components by mistake.
```